src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -68,6 +68,11 @@ for further details.
   max-width: calc(780px + (var(--main-area-padding-left-right) * 2));
   padding-left: var(--main-area-padding-left-right);
   padding-right: var(--main-area-padding-left-right);
+
+  @media (--dev-dot-tablet-down) {
+    padding-left: 0;
+    padding-right: 0;
+  }
```

Comment:
Based on the figma designs, there's only 24px of horizontal space on either side of the main content area for mobile. `.main` felt like the appropriate place to remove spacing from to make this happen.